src/lib/pages/schedules.svelte

```diff
@@ -59,66 +47,69 @@
     emptyStateMessage={translate('schedules.empty-state-title')}
     fallbackErrorMessage={translate('schedules.error-message-fetching')}
   >
-    <header
-      class="flex flex-row justify-between gap-2"
-      slot="header"
-      let:visibleItems
+    <caption class="sr-only" slot="caption"
+      >{translate('common.schedules')}</caption
     >
-      <div>
-        <h1 class="flex flex-col gap-0 md:flex-row md:items-center md:gap-2">
-          <SchedulesCount />
-        </h1>
+
+    <div class="flex flex-col gap-4" slot="header" let:visibleItems>
+      <h1 class="flex flex-col gap-0 md:flex-row md:items-center md:gap-2">
+        <SchedulesCount />
+      </h1>
+
+      <div class="flex flex-row flex-wrap justify-between gap-2">
+        {#if visibleItems.length}
+          <div class="w-96">
+            <Input
+              icon="search"
+              type="search"
+              label={translate('schedules.name')}
+              labelHidden
+              id="schedule-name-filter"
+              placeholder={translate('schedules.name')}
+              clearable
+              clearButtonLabel={translate('common.clear-input-button-label')}
+              bind:value={search}
+              on:submit={noop}
+            />
+          </div>
+        {/if}
+        {#if !createDisabled && visibleItems.length}
+          <Button
+            data-testid="create-schedule"
+            href={routeForScheduleCreate({ namespace })}
+            disabled={!writeActionsAreAllowed()}
+          >
+            {translate('schedules.create')}
+          </Button>
+        {/if}
       </div>
-      {#if !createDisabled && visibleItems.length}
-        <Button
-          data-testid="create-schedule"
-          href={routeForScheduleCreate({ namespace })}
-          disabled={!writeActionsAreAllowed()}
-        >
-          {translate('schedules.create')}
-        </Button>
-      {/if}
-    </header>
-    <svelte:fragment slot="action-top-left" let:visibleItems>
-      {#if visibleItems.length}
-        <div class="w-96">
-          <Input
-            icon="search"
-            type="search"
-            label={translate('schedules.name')}
-            labelHidden
-            id="schedule-name-filter"
-            placeholder={translate('schedules.name')}
-            clearable
-            clearButtonLabel={translate('common.clear-input-button-label')}
-            bind:value={search}
-            on:submit={noop}
-          />
-        </div>
-      {/if}
-    </svelte:fragment>
-    <SchedulesTable>
-      {#each filteredSchedules(visibleItems) as schedule}
-        <SchedulesTableRow {schedule} />
-      {:else}
-        <TableRow>
-          <td class="hidden xl:table-cell" />
-          <td class="hidden xl:table-cell" />
-          <td colspan="3">
+    </div>
+
+    <tr slot="headers" class="text-left">
+      <th class="px-2">{translate('common.status')}</th>
```

Comment:
We may want to consider adding this `px-2` into the PaginatedTable component styles